webapp/locales/es.json

```diff
@@ -119,6 +173,113 @@
     "tag": "Etiqueta ::: Etiquetas",
     "name": "Nombre",
     "loadMore": "cargar más",
-    "loading": "cargando"
+    "loading": "cargando",
+    "reportContent": "Report"
+  },
+  "actions": {
+    "loading": "cargamento",
+    "loadMore": "cargar más",
+    "create": "Crear",
+    "save": "Guardar",
+    "edit": "Edite",
+    "delete": "Delete",
+    "cancel": "Cancelar"
+  },
+  "moderation": {
+    "name": "Moderación",
+    "reports": {
+      "empty": "Felicitaciones, nada que moderar.",
+      "name": "Informes",
+      "submitter": "comunicado por",
+      "disabledBy": "desactivado por"
+    }
+  },
+  "disable": {
+    "submit": "Desactivar",
+    "cancel": "Cancelar",
+    "success": "Discapacitado con éxito",
+    "user": {
+      "title": "Desactivar usuario",
+      "type": "Usuario",
+      "message": "¿Realmente quieres deshabilitar el usuario \"<b>{name}</b>\"?"
+    },
+    "contribution": {
+      "title": "Deshabilitar contribución",
+      "type": "Contribución",
+      "message": "¿Realmente quieres deshabilitar la contribución \"<b>{name}</b>\"?"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-07T22:40:13Z" title="Saturday, June 8th 2019, 12:40:13 am +02:00">Jun 8, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -119,6 +173,113 @@
     "tag": "Etiqueta ::: Etiquetas",
     "name": "Nombre",
     "loadMore": "cargar más",
-    "loading": "cargando"
+    "loading": "cargando",
+    "reportContent": "Report"
+  },
+  "actions": {
+    "loading": "cargamento",
+    "loadMore": "cargar más",
+    "create": "Crear",
+    "save": "Guardar",
+    "edit": "Edite",
+    "delete": "Delete",
+    "cancel": "Cancelar"
+  },
+  "moderation": {
+    "name": "Moderación",
+    "reports": {
+      "empty": "Felicitaciones, nada que moderar.",
+      "name": "Informes",
+      "submitter": "comunicado por",
+      "disabledBy": "desactivado por"
+    }
+  },
+  "disable": {
+    "submit": "Desactivar",
+    "cancel": "Cancelar",
+    "success": "Discapacitado con éxito",
+    "user": {
+      "title": "Desactivar usuario",
+      "type": "Usuario",
+      "message": "¿Realmente quieres deshabilitar el usuario \"<b>{name}</b>\"?"
+    },
+    "contribution": {
+      "title": "Deshabilitar contribución",
+      "type": "Contribución",
+      "message": "¿Realmente quieres deshabilitar la contribución \"<b>{name}</b>\"?"
```


Whaaaat, this is possible? I didn't know you can add variables to a translation.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-06-08T04:56:06Z" title="Saturday, June 8th 2019, 6:56:06 am +02:00">Jun 8, 2019</time>_

---


ahhh ... there is a mistake ... the B element closes but it does not open ... 
an error which e.g. is not recognized!

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-14T00:05:59Z" title="Friday, June 14th 2019, 2:05:59 am +02:00">Jun 14, 2019</time>_

---


what do you mean @ogerly... I needed to add this `<b></b>` to a `v-html` for it to get converted to bold, instead of it being read literally, but maybe that is not what you mean at all.

@roschaefer, I was also pleasantly surprised to see this!! and it's super handy when you need to pass in computed values, like the `commentsCount`, `contributionsCount`